.circleci/src/jobs/check.yml

```diff
@@ -17,7 +17,7 @@ steps:
       command: yarn prettier:check
   - run:
       name: TS check
-      command: yarn ts:check
+      command: node --max-old-space-size=1536 ./node_modules/.bin/tsc --build
```

Comment:
I ran into memory problems with `ts:check` so I had to increase the node js heap size.

Reply:
So can you please create a command in package.json for this, similar to "lint:ci" we chould introduce a "ts:check:ci"